serverless.yml

```diff
@@ -50,6 +50,7 @@ functions:
       SMS_API_URL: ${ssm:/shared-plan/SMS_API_URL}
       PLANS_TABLE_NAME: ${self:provider.environment.PLANS_TABLE}
       NEXT_PUBLIC_API_URL: https://${self:custom.aliases.${self:provider.stage}}/api
+      NEXT_PUBLIC_URL: ${self:provider.environment.NEXT_PUBLIC_URL}
```

Comment:
Is this the same value as: `https://${self:custom.aliases.${self:provider.stage}}`? If so we could use that and save ourselves from having to set another environment variable in Circle?

Reply:
💯 You already fixed this!